test/unit/notify/send-emails.test.js

```diff
@@ -118,9 +127,15 @@ describe('send emails', () => {
     expect(formatEmailAddresses).toHaveBeenCalledTimes(1)
   })
 
-  test('should not call formatEmailAddress when emailAddress is an array', async () => {
+  test('should call formatEmailAddress when emailAddress is an array', async () => {
```

Comment:
Would move all `should call formatEmailAddress` before `should call sendEmail` given it's called before in the file